array_api_compat/dask/array/_aliases.py

```diff
@@ -228,10 +230,84 @@ def _isscalar(a):
 
     return astype(da.minimum(da.maximum(x, min), max), x.dtype)
 
-# exclude these from all since dask.array has no sorting functions
-_da_unsupported = ['sort', 'argsort']
 
-_common_aliases = [alias for alias in _aliases.__all__ if alias not in _da_unsupported]
+def sort(
+    x: Array, /, *, axis: int = -1, descending: bool = False, stable: bool = True
+) -> Array:
+    """
+    Array API compatibility layer around the lack of sort() in Dask.
+
+    Warnings
+    --------
+    This function temporarily rechunks the array along `axis` to a single chunk.
+    This can be extremely inefficient and can lead to out-of-memory errors.
+
+    See the corresponding documentation in the array library and/or the array API
+    specification for more details.
+    """
+    return _sort_argsort("sort", x, axis=axis, descending=descending, stable=stable)
+
+
+def argsort(
+    x: Array, /, *, axis: int = -1, descending: bool = False, stable: bool = True
+) -> Array:
+    """
+    Array API compatibility layer around the lack of argsort() in Dask.
+
+    See the corresponding documentation in the array library and/or the array API
+    specification for more details.
+
+    Warnings
+    --------
+    This function temporarily rechunks the array along `axis` into a single chunk.
+    This can be extremely inefficient and can lead to out-of-memory errors.
+    """
+    return _sort_argsort("argsort", x, axis=axis, descending=descending, stable=stable)
+
+
+def _sort_argsort(
+    func: Literal["sort", "argsort"],
+    x: Array,
+    /,
+    *,
+    axis: int,
+    descending: bool,
+    stable: bool,
+) -> Array:
+    """
+    Implementation of sort() and argsort()
+
+    TODO Implement sort and argsort properly in Dask on top of the shuffle subsystem.
+    """
+    if axis < 0:
+        axis += x.ndim
+    rechunk = False
+    if x.numblocks[axis] > 1:
+        rechunk = True
+        # Break chunks on other axes in an attempt to keep chunk size low
+        x = x.rechunk({i: -1 if i == axis else "auto" for i in range(x.ndim)})
+    meta_xp = array_namespace(x._meta)
```

Comment:
FIXME: I couldn't write a test that actually uses a meta_xp other than numpy, because `da.from_array(array_api_strict.asarray(0))` doesn't work in Dask yet.